root/static/scripts/release-editor/trackParser.js

```diff
@@ -485,17 +485,19 @@ trackParser.customDelimiterError = debounce(function () {
 })
 
 function optionCookie(name, defaultValue, checkbox=true) {
-    var existingValue = getCookie(name);
-
-    if (checkbox) {
-      var observable = ko.observable(
-          defaultValue ? existingValue !== "false" : existingValue === "true"
-      );
-    } else {
-      var observable = ko.observable(
-          existingValue ? existingValue : defaultValue
-      );
-    }
+    const existingValue = getCookie(name);
+
+    const observable = checkbox  
+          ? ko.observable(
```

Comment:
Just a small thing, but this all appears to be indented 6 spaces instead of 4. Can you fix that? :)

Also, below we can replace `existingValue ? existingValue : defaultValue` with `existingValue || defaultValue` so that it can all fit in one line, like `: ko.observable(existingValue || defaultValue)`. :)